dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -55,7 +55,7 @@ public async Task OpenAITestAsync(string prompt, string expectedAnswerContains)
                 setAsDefault: true)
             .Build();
 
-        IDictionary<string, ISKFunction> skill = TestHelpers.GetSkills(target, "ChatSkill");
+        IDictionary<string, ISKFunction> skill = TestHelpers.GetSkills(target, "ChatPlugin");
```

Comment:
So looks like we are mixing the term `skills` and `plugins`, any plan on consolidating them?